GameData/ROCapsules/PartConfigs/Vostok/VostokCapsule.cfg

```diff
@@ -1,193 +1,203 @@
-	//  ============================================================================
-	
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = Pods
+	subcategory = 0
 	title = Vostok Descent Module
 	manufacturer = OKB-1
 	description = The Vostok SA Descent Module was the first spacecraft to carry humans into space when Yuri Gagarin flew on April 12, 1961. The SA stands for Spuskaemiy Apparat (Descent System) and was designed as a ballistic reentry capsule. [Diameter 2.3 m]
-	
-	mass = 2.325
-	
-	category = Pods
-	subcategory = 0
-	
 	tags = vostok, gagarin, soviet, capsule, crew, descent
-	
-	TechRequired = unlockParts
-	cost = 150
-	entryCost = 1
-	
+
 	vesselType = Ship
 	stagingIcon = COMMAND_POD
 	CrewCapacity = 1
-	
-	INTERNAL
-	{
-		name = SoyuzPod
-	}
-	
-	//  ============================================================================
+	bulkheadProfiles = size1, size2
+
+	//	============================================================================
 	//	Thermo, Durability
-	//  ============================================================================
-		
+	//	============================================================================
+
+	mass = 2.325
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 2
 	crashTolerance = 14
-	maxTemp = 2400
-	skinMaxTemp = 3600
-	emissiveConstant = 0.4 // not too absorptive for reentry
+	breakingForce = 250
+	breakingTorque = 250
+	fuelCrossFeed = True
+
+	heatShieldDiameter	= 2.3
+	heatShieldTag = LEO
+	resetHeatShieldAblator = true
+	resetHeatShieldMass = false
+	emissiveConstant = 0.6
 	thermalMassModifier = 1.0
 	skinMassPerArea = 4
-	
-	MODULE:NEEDS[!DeadlyReentry]
-	{
-		name = ModuleAblator
-		ablativeResource = Ablator
-		lossExp = -8000
-		lossConst = 0.1
-		pyrolysisLossFactor = 25000
-		ablationTempThresh = 500
-		reentryConductivity = 0.01
-	}
-	
-	MODULE:NEEDS[DeadlyReentry]
-	{
-		name = ModuleHeatShield
-		ablativeResource = Ablator
-		lossExp = -8000
-		lossConst = 0.1
-		pyrolysisLossFactor = 25000
-		ablationTempThresh = 500
-		reentryConductivity = 0.01
-		depletedMaxTemp = 1200
-	}
-	
-	RESOURCE
-	{
-		name = Ablator
-		amount = 225
-		maxAmount = 225
-	}
-	
-	//  ============================================================================
+
+	//	============================================================================
 	//	Modules and Resources
-	//  ============================================================================
-	
+	//	============================================================================
+
 	MODULE
 	{
 		name = ModuleCommand
 		minimumCrew = 0
 		RESOURCE
 		{
 			name = ElectricCharge
-			rate = 0.25 // A little more than half the Mercury Capsule as we will put it in the SM
+			rate = 0.25
 		}
 	}
-	
-	// Cannot EVA from Here
-	MODULE:NEEDS[RP-0]
+
+	MODULE
 	{
-		name = ModuleNoEVA
```

Comment:
If this requires the airlock part, think this NoEVA needs to remain?

Reply:
Nope its already built in to RP-0